### DISCUSSION TITLE
Window Coordinates

### DISCUSSION THREAD

**Discussion by 24unix** [rxhanson/RectanglePro-Community#414] [Q&A]
Dear Rectangle Pro Team,

I recently started using Rectangle Pro after transitioning from Hyper Dock, especially following compatibility issues with Hyper Dock in macOS Ventura.
For main window management I use Yabai, but Rectangles feature to move/resize windows without clicking the mouse is great.

However, there's one feature that I miss and would greatly appreciate in Rectangle Pro. When moving or resizing windows, it would be incredibly helpful to have an optional overlay display that shows the current coordinates and dimensions of the window. This feature would be helpful while developing macOS apps.

I understand that feature development takes time and resources, but I believe this addition could benefit many users who value meticulous window management. Thank you for considering my suggestion, and I look forward to future updates that will continue to enhance the user experience of Rectangle Pro.

Best regards,
  MIcha

**Comment by rxhanson** [comment]
Thanks for the feedback! I've had this come up once before and thought about building in some kind of ruler overlay. My initial conclusion was that this would be better served as a single purpose separate app, where you can easily toggle the ruler overlay. I won't say that I'll never add this in to Rectangle Pro, but the separate app seems optimal to me.
